bigframes/session/loader.py

```diff
@@ -529,6 +529,9 @@ def read_gbq_query(
             configuration=configuration,
         )
 
+        if self._metrics is not None:
```

Comment:
I missed earlier, is the query_job guaranteed to have finished at this point? We should wait for the job to finish, like we do in other places before calling `count_job_stats`.